change/@ni-nimble-components-dbd8f845-12ff-44bd-b2de-86145d22572c.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Add nowrap to button content styling",
```

Comment:
>It looks like Chromatic is reporting that the icon button snapshot is a bit smaller than before. Perhaps, the snapshot was allocating space for the hidden label, and that size changed?

It's definitely weird but I poked around in the actual DOM and didn't see any issues so I'm fine to accept the diffs.